onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -1289,7 +1289,7 @@ void addObjectMethods(py::module& m, Environment& env) {
         int type_num = dtype->type_num;
         Py_DECREF(dtype);
 
-        OrtMemoryInfo info(GetDeviceName(device), OrtDeviceAllocator, device);
+        OrtMemoryInfo info(GetDeviceName(device), OrtDeviceAllocator, device, device.Id());
```

Comment:
>device.Id() [](start = 78, length = 11)

Seems like a bug that would be caught with multi-GPU CI. Some following up questions:
1. Should we change the OrtMemoryInfo ctor to not take default arguments of device_id == 0?
2. Do we have test coverage on this?

Reply:
I think we should merge these two fields.

---
In reply to: [516222469](https://github.com/microsoft/onnxruntime/pull/5663#discussion_r516222469) [](ancestors = 516222469)

Reply:
I have a multi-GPU daily pipeline. But I guess it didn't cover this code.

Reply:
@liqunfu Do we have CI tests in Python that uses multi-GPU?

Reply:
@hectorsvc set up a multi gpu inference test pipeline in the past.

Reply:
Yes. https://aiinfra.visualstudio.com/Lotus/_build/results?buildId=136907&view=results